shapefile.py

```diff
@@ -201,6 +201,11 @@ def __init__(self, shapeType=NULL, points=None, parts=None, partTypes=None):
 
     @property
     def __geo_interface__(self):
+
+        """  Case parts or points is empty """
+        if not self.parts or not self.points:
+            return False

```

Comment:
There doesn't seem to be an empty geometry type defined in GeoJSON, but it does state that feature objects without a geometry can be specified with JSON null value, which would be None rather than False in Python. 

From the previous link:

> A feature object must have a member with the name "geometry". The value of the geometry member is a geometry object as defined above or a JSON null value. 

I also think None makes more sense since there literally is "none" geometry object.